### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by almuitur**

### CODE REVIEW

File: staff/almu-itur/app-template/about.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            <p>Lorem ipsum dolor sit amet consectetur adipiscing elit a, bibendum est parturient sociosqu facilisis eu penatibus lobortis semper, condimentum purus eget ligula odio etiam dictum. Massa augue odio penatibus lacus nec litora nullam nunc, fringilla nulla venenatis porttitor phasellus aptent curabitur, facilisis vel tempor posuere potenti etiam imperdiet. Turpis eget velit volutpat tristique class cum, natoque ut lacus rutrum risus ac praesent, vel hac sociis magnis platea.</p>
+            <p>Lorem ipsum dolor sit amet consectetur adipiscing elit a, bibendum est parturient sociosqu facilisis eu penatibus lobortis semper, condimentum purus eget ligula odio etiam dictum. Massa augue odio penatibus lacus nec litora nullam nunc, fringilla nulla venenatis porttitor phasellus aptent curabitur, facilisis vel tempor posuere potenti etiam imperdiet. Turpis eget velit volutpat tristique class cum, natoque ut lacus rutrum risus ac praesent, vel hac sociis magnis platea.</p>
+        </article>
```
Comment (aleixcam) [inline staff/almu-itur/app-template/about.html:35]: may you want to use a section here?

File: staff/almu-itur/app-template/news.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <article>
+            <header>
+                <h1>Main heading</h1>
+                <h3>Secondary heading</h3>
+                <time datetime='2018-1-1'>January 1st</time>
+            </header>
+            <picture srcset="image.jpg"></picture>
+            <p>Lorem Ipsum dolor set amet....</p>
+            <figure>
+                <img src="image-article.jpg">
```
Comment (gbaste) [inline staff/almu-itur/app-template/news.html:38]: The alt attribute is required.

[alt information](https://www.w3schools.com/TAGS/att_img_alt.asp)

File: staff/almu-itur/app-template/about.html (hunk 1 above)
Comment (manuelbarzi) [reply staff/almu-itur/app-template/about.html:35]: agree... article has no sense here, please, RTFM

**Comment by manuelbarzi** [comment]
well done, in general. you have projected a template with nav in important views, that's a good thing, as all important pages should have, for instance, `nav` element for the different pages of the site. good

File: staff/almu-itur/app-template/about.html (hunk 1 above)
Comment (almuitur) [reply staff/almu-itur/app-template/about.html:35]: Ok

File: staff/almu-itur/app-template/news.html (hunk 2 above)
Comment (almuitur) [reply staff/almu-itur/app-template/news.html:38]: Ok

**Pull request merged**